src/VirtoCommerce.DemoSolutionFeaturesModule.Data/VirtoCommerce.DemoSolutionFeaturesModule.Data.csproj

```diff
@@ -23,7 +22,7 @@
     </PackageReference>
     <PackageReference Include="VirtoCommerce.CartModule.Data" Version="3.7.0" />
     <PackageReference Include="VirtoCommerce.CustomerModule.Data" Version="3.9.0" />
-    <PackageReference Include="VirtoCommerce.OrdersModule.Data" Version="3.9.0" />
+    <PackageReference Include="VirtoCommerce.OrdersModule.Data" Version="3.11.0" />
```

Comment:
Set same dependent version in the module.manifest

Reply:
Fixed